src/components/Image/index.tsx

```diff
@@ -0,0 +1,15 @@
+import PropTypes from 'prop-types'
+import React, { FunctionComponent } from 'react'
+import Box from '../Box'
+
+const Image: FunctionComponent<HTMLImageElement & XStyledProps> = ({
+  src,
+  ...props
+  // @ts-expect-error Box expose Element, not ImgElement
+}) => <Box as="img" src={src} {...props} />
```

Comment:
Props are `HTMLAttributes<HTMLImageElement>` & you may just forward all props:
```suggestion
const Image: FunctionComponent<HTMLAttributes<HTMLImageElement> & XStyledProps> = (props) => <Box as="img" src={src} {...props} />
```

Reply:
Corrected thanks :)

Reply:
Corrected thanks :)